index.coffee

```diff
@@ -69,18 +12,25 @@ module.exports =
       default: ''
 
   activate: ->
-    prefix = 'linter-rubocop.'
     @subscriptions = new CompositeDisposable
-    @subscriptions.add atom.config.observe "#{prefix}executablePath",
-      (args) => @executablePath = if args then args.split ' ' else ['rubocop']
-    @subscriptions.add atom.config.observe "#{prefix}additionalArguments",
-      (args) => @additionalArguments = if args then args.split ' ' else []
+    @subscriptions.add atom.config.observe('linter-rubocop.executablePath', (executablePath) =>
+      @executablePath = executablePath
```

Comment:
Yep we discussed on slack, I'll be adding the space delimited extra commands again when I work on this